Description: Correct the number of args given to printf (zero-indexed/one-indexed confusion).
Forwarded: yes
Author: Rowan Thorpe <rowan@rowanthorpe.com>
Last-Update: 2013-09-26
--- a/src/sflowtool.c
+++ b/src/sflowtool.c
@@ -517,7 +517,7 @@ static char *printAddress(SFLAddress *address, char *buf, int bufLen) {
     u_char *b = address->address.ip_v6.addr;
     // should really be: snprintf(buf, buflen,...) but snprintf() is not always available
     sprintf(buf, "%02x%02x:%02x%02x:%02x%02x:%02x%02x:%02x%02x:%02x%02x:%02x%02x:%02x%02x",
-	    b[0],b[1],b[2],b[3],b[4],b[5],b[6],b[7],b[8],b[9],b[10],b[11],b[12],b[13],b[14],b[15],b[16]);
+	    b[0],b[1],b[2],b[3],b[4],b[5],b[6],b[7],b[8],b[9],b[10],b[11],b[12],b[13],b[14],b[15]);
   }
   return buf;
 }
